css/styles.css

```diff
@@ -0,0 +1,155 @@
+/* stylelint-disable */
+* {
+    margin: 0;
+    padding: 0;
+    text-decoration: none;
+    text-align: center;
+  }
+ 
+.body h1 {
+    color: #172b4d;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+  
+.menu {
+    display: flex;
+    gap: 30px;
+    justify-content: center;
+    list-style: none;
+    margin: 20px;
+  }
+  
+  .h-menu {
+    display: none;
+  }
+  
+.body h2 {
+    color: #30a02c;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    margin-top: 10px;
+  }
+  
+.body p {
+    color: #172b4d;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    margin-top: 40px;
+  }
+  
+.icon {
+    margin-top: 40px;
+  }
+  
+.icon ul {
+    display: flex;
+    justify-content: center;
+    list-style: none;
+  }
+  
+.icon a {
+    margin: 10px;
+    color: #505f79;
+    font-size: 20px;
+  }
+  
+.icon a:hover {
+    cursor: pointer;
+    color: blue;
+  }
+  
+@media (max-width: 770px) {
+    .h-menu {
+      display: flex;
+      justify-content: space-between;
+      margin-top: 20px;
+      margin-right: 20px;
+      margin-left: 10px;
+      height: 60px;
+    }
+  
+    .body {
+      background-image: url(./images/header-img.png);
```

Comment:
- [ ] Please make sure that the `background-image` is set for the mobile version too (not just for the desktop version) because it is in the Figma design
 Your design:
![headline2](https://user-images.githubusercontent.com/95412943/219703279-62044668-1fe9-48ad-ad29-bd7c060356ab.PNG)

Figma design:
![headline](https://user-images.githubusercontent.com/95412943/219703353-ac6fde40-8197-426b-bfd8-1177c72efdfb.PNG)

Reply:
Done